server.js

```diff
@@ -3,6 +3,15 @@ const express = require("express");
 const PORT = 3001;
 const app = express();
 
+app.get("/", (req, res) => {
+  return res.json({ ping: "pong" });
+});
+
+app.get("/random", (req, res) => {
```

Comment:
might be cool to have another random route that allows for a min and max to come in through the url?

Reply:
thanks! updated.